modules/checkbox/react/lib/Checkbox.tsx

```diff
@@ -179,24 +183,27 @@ const CheckboxInput = styled('input')<CheckboxProps>(
   }
 );
 
-const CheckboxBackground = styled('div')<CheckboxProps>({
-  alignItems: 'center',
-  backgroundColor: colors.frenchVanilla100,
-  borderRadius: borderRadius.s,
-  border: '1px solid ' + inputColors.border,
-  boxSizing: 'border-box',
-  display: 'flex',
-  height: checkboxHeight,
-  justifyContent: 'center',
-  padding: '0px 2px',
-  pointerEvents: 'none',
-  position: 'absolute',
-  transition: 'border 200ms ease, background 200ms',
-  width: checkboxWidth,
-});
+const CheckboxBackground = styled('div')<CheckboxProps>(
+  DirectionProvider(direction, {
+    alignItems: 'center',
+    backgroundColor: colors.frenchVanilla100,
+    borderRadius: borderRadius.s,
+    border: '1px solid ' + inputColors.border,
+    boxSizing: 'border-box',
+    display: 'flex',
+    height: checkboxHeight,
+    justifyContent: 'center',
+    marginTop: direction === Direction.RTL ? '-22px' : 0,
```

Comment:
This is to show how we could specify styles to a direction. This is to fix the checkbox in RTL but unfortunately it only works in Chrome (Safari works without the need for this fix), so Checkbox will need some kind of refactoring I think.